tasks.py

```diff
@@ -180,7 +181,10 @@ def deploy(ctx, space=None, branch=None, login=None, yes=False):
     # Target space
     ctx.run('cf target -o fec-beta-fec -s {0}'.format(space), echo=True)
 
-    run_migrations(ctx, space)
+    print("\nMigrating database...")
+    jdbc_url = to_jdbc_url(os.getenv('FEC_MIGRATOR_SQLA_CONN_{0}'.format(space.upper())))
+    run_migrations(ctx, jdbc_url)
+    print("Database migrated\n")
```

Comment:
Should these be `logger.info(...)` statements instead?  Or is this to account for output in CircleCI?  Would this be a case of using both?

Reply:
This is a mere message to the user about stages pf completion. I felt there was no need for the kind of detail and configurability that `logging` provides. I wanted to reduce clutter.

Reply:
Ah okay - seeing as how this'll show up in our consoles locally or in the CircleCI output, which is all we're concerned about, that makes sense!